src/Quizzer.API.BL/Facades/QuizFacade.cs

```diff
@@ -0,0 +1,167 @@
+﻿using AutoMapper;
+using Quizzer.API.BL.Models;
+using Quizzer.API.DAL.Entities;
+using Quizzer.API.DAL.Repository.Interfaces;
+using Quizzer.API.DAL.UnitOfWork;
+using Quizzer.Extensions;
+using System.Linq.Expressions;
+
+namespace Quizzer.API.BL.Facades {
+    public class QuizFacade : FacadeBase<QuizEntity, QuizDetailModel> {
+        public QuizFacade(IUnitOfWorkFactory unitOfWorkFactory, IMapper mapper) : base(unitOfWorkFactory, mapper) {
+        }
+
+        public new Guid Save(QuizDetailModel model) {
+            if (model.StartTime < model.EndTime) {
```

Comment:
maybe the `<=` operator would be better because if they are same, the quiz will actually never start. Also there could be check that quiz will start later than now - so there will not be an option to set start of quiz to year 1864.

Reply:
I think It should be switched, to this: `>` because then start is later than end. I also added the suggested now time for checking.